src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/VisitSessionController.kt

```diff
@@ -85,8 +85,13 @@ class VisitSessionController(
       example = "28",
     )
     max: Int?,
+    @Parameter(
+      description = "Username for the user making the request. Optional, ignored if not passed in.",
```

Comment:
Can we briefly explain what passing it in does?

E.g.

description = "Username for the user making the request. Used to exclude users pending applications from session capacity count. Optional, ignored if not passed in."

Reply:
Thanks, good suggestion! Updated